src/pretix/presale/context.py

```diff
@@ -16,6 +17,17 @@ def contextprocessor(request):
     }
     _html_head = []
     _footer = []
+
+    with language(request.LANGUAGE_CODE):
```

Comment:
I'm unsure whether this context manager is really necessary, it should not be, but it surely does not hurt.